From 72f9fad5d973d45bd8bdc455d94499ecfafbd722 Mon Sep 17 00:00:00 2001
From: Marcelo Leitner <mleitner@redhat.com>
Date: Tue, 21 Oct 2014 16:16:52 -0400
Subject: [net] netfilter: Add missing vmalloc.h include to nft_hash.c

Message-id: <20141021161652.22888.64624.stgit@localhost.localdomain>
Patchwork-id: 97922
O-Subject: [RHEL7 net PATCH 027/124] netfilter: Add missing vmalloc.h include to nft_hash.c
Bugzilla: 1148041 1155088
RH-Acked-by: Florian Westphal <fwestpha@redhat.com>

Backport of upstream commit: 3ab428a4c5ad929f93f9742d2a222ae33c242c3d

 commit 3ab428a4c5ad929f93f9742d2a222ae33c242c3d
 Author: David S. Miller <davem@davemloft.net>
 Date:   Tue Mar 18 23:12:02 2014 -0400

    netfilter: Add missing vmalloc.h include to nft_hash.c

    Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Marcelo Ricardo Leitner <mleitner@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/netfilter/nft_hash.c b/net/netfilter/nft_hash.c
index 6a1acde..3b1ad87 100644
--- a/net/netfilter/nft_hash.c
+++ b/net/netfilter/nft_hash.c
@@ -14,6 +14,7 @@
 #include <linux/list.h>
 #include <linux/jhash.h>
 #include <linux/netlink.h>
+#include <linux/vmalloc.h>
 #include <linux/netfilter.h>
 #include <linux/netfilter/nf_tables.h>
 #include <net/netfilter/nf_tables.h>
-- 
1.7.1